terminus-ui/option/src/optgroup.component.ts

```diff
@@ -138,7 +138,7 @@ export class TsSelectOptgroupComponent {
     private changeDetectorRef: ChangeDetectorRef,
     // NOTE: Useful for testing
     // tslint:disable-next-line
```

Comment:
The next line isn't used by the component itself, but used in testing.

Reply:
Ahh ok, so I'm guessing the TSLint error we are silencing is the 'no-unused-..`. Can we a) disable a specific rule for that line rather than all of tslint, & b) Add a better note.. something like 'ElementRef is useful for testing but not used within this file' etc.

(I know that poor comment is likely from me 🤦‍♂)